src/spec/template_spec.js

```diff
@@ -0,0 +1,279 @@
+          "bodyType": "rawHtml",
+          "_links": []
+        });
+
+        $httpBackend.whenGET(/\/accounts\/[\d|-]*\/templates\/[\w|-]*\/body/).respond(200, {
+          "name": "test",
+          "html": "test",
+          "_links": []
+        });
+          
+        $httpBackend.whenGET(/\/accounts\/[\w|-]*\/status\/limits/).respond(200, {
+          "noLimits": true,
+          "dkimConfigurationRequired": false,
+          "domainConfigurationRequired": false,
+          "_links": []
+        });
+        
+        $httpBackend.whenGET(/\/accounts\/[\d|-]*\/templates/).respond(200, {
+          "items": [
+            {
+              "from_name": "test name",
+              "from_email": "test@test.test",
+              "subject": "test subject",
+              "body": "Template body test",
+              "name": "Test template",
+              "id": "1169ab26-78dc-4474-a8c2-e1f02f6c05ed",
+              "last_updated": "2018-07-12T15:45:19Z",
+              "created_at": "2018-07-12T13:27:40Z",
+              "bodyType": "rawHtml",
+              "_links": []
+            }
+          ],
+          "pageSize": 200,
+          "itemsCount": 1,
+          "currentPage": 1,
+          "pagesCount": 1,
+          "_links": []
+        });
+
+        $httpBackend.whenPOST(/\/accounts\/[\d|-]*\/templates/).respond(200, {
+          "createdResourceId": "1169ab26-78dc-4474-a8c2-e1f02f6c05ed",
+          "message": "Template successfully created",
+          "_links": []
+        });
+
+        $httpBackend.whenPUT(/\/accounts\/[\d|-]*\/templates\/[\w|-]*/).respond(200, {
+          "message": "Template successfully updated",
+          "_links": []
+        });
+
+        $httpBackend.whenPUT(/\/accounts\/[\d|-]*\/templates\/[\w|-]*\/body/).respond(200, {
+          "message": "Template successfully updated",
+          "_links": []
+        });
+      }
+    ));
+  }
+
+  function setupCreateTemplateResponse() {
+    browser.addMockModule('descartableModule3', () => angular
+      .module('descartableModule3', ['ngMockE2E'])
+      .run($httpBackend => {
+        
+
+        $httpBackend.whenPUT(/\/accounts\/[\d|-]*\/templates\/[\w|-]*\/body/).respond(200, {
+          "message": "Template successfully updated",
+          "_links": []
+        });
+      }
+    ));
+  }
+  it('should show template form texts in edit mode in english', () => {
+    // Arrange
+    beginAuthenticatedSession();
+    browser.get('/#/templates/32e93fa4-c8f4-467a-a2bb-da95f72a27b4');
+    setupTemplateResponse();
+
+    var templatePage = new TemplatePage();
+    var expectedTitleText = "Template Info";
+    var expectedTemplateNameText = "Template name";
+    var expectedFromEmailText = "From Email";
+    var expectedFromNameText = "From Name";
+    var expectedSubjectText = "Subject";
+    var expectedTemplateEditorTitle = "Template Editor";
+    var expectedTemplateEditorDescription = "Here's where you can create and edit your custom template.";
```

Comment:
I think that testing literals is not a good idea.

Reply:
@mvillaseco do you know if it's possible to include en-translation.js / es-translation in tests?

Reply:
@andresmoschini do you think this test is necessary? I don't found a better approach to test it.

Reply:
I will remove this test